docker/go-carbon/go-carbon.conf

```diff
@@ -0,0 +1,271 @@
+[common]
+# Run as user. Works only in daemon mode
+user = ""
+# Prefix for store all internal go-carbon graphs. Supported macroses: {host}
+graph-prefix = "carbon.agents.{host}"
+# Endpoint for store internal carbon metrics. Valid values: "" or "local", "tcp://host:port", "udp://host:port"
+metric-endpoint = "local"
+# Interval of storing internal metrics. Like CARBON_METRIC_INTERVAL
+metric-interval = "1m0s"
+# Increase for configuration with multi persister workers
+max-cpu = 4
+
+[whisper]
+data-dir = "/data/graphite/whisper"
+# http://graphite.readthedocs.org/en/latest/config-carbon.html#storage-schemas-conf. Required
+schemas-file = "/etc/storage-schemas.conf"
+# http://graphite.readthedocs.org/en/latest/config-carbon.html#storage-aggregation-conf. Optional
+aggregation-file = "/etc/storage-aggregation.conf"
+# Worker threads count. Metrics sharded by "crc32(metricName) % workers"
+workers = 8
+# Limits the number of whisper update_many() calls per second. 0 - no limit
+max-updates-per-second = 0
+# Sparse file creation
+sparse-create = false
+enabled = true
+
+[cache]
+# Limit of in-memory stored points (not metrics)
+max-size = 1000000
+# Capacity of queue between receivers and cache
+# Strategy to persist metrics. Values: "max","sorted","noop"
+#   "max" - write metrics with most unwritten datapoints first
+#   "sorted" - sort by timestamp of first unwritten datapoint.
+#   "noop" - pick metrics to write in unspecified order,
+#            requires least CPU and improves cache responsiveness
+write-strategy = "max"
+
+[udp]
+listen = ":2003"
+enabled = true
+# Enable optional logging of incomplete messages (chunked by max UDP packet size)
+log-incomplete = false
+# Optional internal queue between receiver and cache
+buffer-size = 0
+
+[tcp]
+listen = ":2003"
+enabled = true
+# Optional internal queue between receiver and cache
+buffer-size = 0
+
+[pickle]
+listen = ":2004"
+# Limit message size for prevent memory overflow
+max-message-size = 67108864
+enabled = true
+# Optional internal queue between receiver and cache
+buffer-size = 0
+
+# You can define unlimited count of additional receivers
```

Comment:
I am pretty sure we can clean up a bit here. These are left as examples in the upstream repo, we probably don't need these comments here.